lib/zendesk_apps_tools/server.rb

```diff
@@ -17,8 +17,13 @@ class Server < Sinatra::Base
           last_mtime = curr_mtime
         end
       end
+      settings.parameters.merge!( { locale: normalize_locale(params['locale']) } )
 
       ZendeskAppsSupport::Package.new(settings.root).readified_js(nil, settings.app_id, "http://localhost:#{settings.port}/", settings.parameters)
     end
+
+    def normalize_locale(locale)
```

Comment:
`locale` could be nil? Can set default here `def normalize_locale(locale = 'en')`.